data/transactions/logic/README.md

```diff
@@ -115,8 +116,8 @@ For two-argument ops, `A` is the previous element on the stack and `B` is the la
 | `\|\|` | A is not zero or B is not zero => {0 or 1} |
 | `shl` | A times 2^B, modulo 2^64 |
 | `shr` | A divided by 2^B |
-| `sqrt` | The largest integer X such that X^2 <= A |
-| `bitlen` | The index of the highest bit in A. If A is a byte-array, it is interpreted as a big-endian unsigned integer |
+| `sqrt` | The largest integer B such that B^2 <= X |
+| `bitlen` | The highest set bit in X. If X is a byte-array, it is interpreted as a big-endian unsigned integer. bitlen of 0 is 0, bitlen of 8 is 4 |
```

Comment:
I'm confused by the change from A/B to X here. Isn't it convention to use A, B, etc. for stack values?

Reply:
I thought so too, but it was pointed out to me that the convention that's actually explained in the docs is that single argument opcodes use X.

Reply:
Huh - good to know